service/controller/clusterapi/v29/resource/tcnp/testdata/case-0-basic-test.golden

```diff
@@ -19,7 +19,7 @@ Resources:
     Properties:
       VPCZoneIdentifier:
         - !Ref PrivateSubnetEuCentral1a
-        - !Ref PrivateSubnetEuCentral1b
+        - !Ref PrivateSubnetEuCentral1c
```

Comment:
That changed because I changed the AZ in the default controller context we use for the unit tests. This matches now what the `MachineDeployment` actually sets.